cmd/admin-server-info.go

```diff
@@ -111,3 +113,34 @@ func getLocalDrivesPerf(endpoints EndpointList, size int64, r *http.Request) mad
 		Perf: dps,
 	}
 }
+
+// getLocalCPUHardware - returns ServerCPUHardwareInfo only for the
+// local endpoints from given list of endpoints
+func getLocalCPUHardware(endpoints EndpointList, r *http.Request) madmin.ServerCPUHardwareInfo {
+	var cpuHardwares []CPUHw.InfoStat
+	seenHosts := set.NewStringSet()
+	for _, endpoint := range endpoints {
+		if seenHosts.Contains(endpoint.Host) {
+			continue
+		}
+		// Only proceed for local endpoints
+		if endpoint.IsLocal {
+			cpuHardware, err := CPUHw.Info()
```

Comment:
```suggestion
			cpuInfo, err := cpuhw.Info()
```

Is better, it is unusual to capitalize package names

Reply:
done.